_posts/2019-02-02-homebrew-2.0.0.md

```diff
@@ -0,0 +1,45 @@
+- [Homebrew's portable Ruby is now built on OS X Mavericks (10.9) for improved performance.](https://github.com/Homebrew/brew/pull/5624)
+- [`brew install` does not try `link` formulae that already have a cask with the same name installed](https://github.com/Homebrew/brew/pull/5613). This avoids link errors in these cases.
+
+Also, if you've not tuned in since 1.0.0, here are the major changes since then:
+
+- [Rubocop-supporting editors (e.g. VS Code, Atom) can get inline `brew audit` information when editing formulae in taps.](https://github.com/Homebrew/brew/pull/1128) This improves the contribution experience.
+- [A `brew tap-new` command is available for creating a new tap with a `README`](https://github.com/Homebrew/brew/pull/1277) [and preconfigured Azure Pipelines configuration](https://github.com/Homebrew/brew/pull/5457) (which seems to provide the most reliable and performant macOS CI for OSS at the time of writing). This eases the creation of taps (third-party repositories).
+- [`brew update-reset` resets all repositories and taps to their upstream versions.](https://github.com/Homebrew/brew/pull/1684) This is useful when debugging `git` issues.
+- [`brew link` state is preserved after `brew install` and `brew upgrade` (including for keg-only formulae)](https://github.com/Homebrew/brew/pull/2961) but unfortunately not the [`brew unlink` state](https://github.com/Homebrew/brew/pull/3660) due to a lack of state. This should allow many keg-only formulae to be used as if they are normal formulae.
+- Homebrew filters all user environment variables. This reduces errors when formulae are built source and allows the removal of many workarounds for niche issues.
+- [`brew postgresql-upgrade-database` upgrades PostgreSQL database data between major versions](https://github.com/Homebrew/homebrew-core/blob/master/cmd/brew-postgresql-upgrade-database.rb). This simplifies upgrades between PostgreSQL versions which previously required a semi-manual process and old version of PostgreSQL to be kept around.
+- The `python` formula was upgraded to Python 3.x and `python@2` formula was added for installing Python 2.7. We initially did not comply with [PEP 394](https://www.python.org/dev/peps/pep-0394/) and this was a mistake. We made `brew install python` and `brew install python@2` PEP 394 compliant and will not change this again until PEP 394 has changed. This allows us to migrate more of our ecosystem to Python 3.
+- We deprecated and archived the all Homebrew organisation taps and created new and versioned formulae in [Homebrew/homebrew-core](https://github.com/Homebrew/homebrew-core). We encourage more niche formulae and formula options to be supported in taps outside the Homebrew organisation. This allows us to focus on providing high-quality formulae that work consistently rather than a long-tail of broken, unsupported, ignored formulae.
```

Comment:
```suggestion
- We deprecated and archived all the Homebrew organisation taps and created new and versioned formulae in [Homebrew/homebrew-core](https://github.com/Homebrew/homebrew-core). We encourage more niche formulae and formula options to be supported in taps outside the Homebrew organisation. This allows us to focus on providing high-quality formulae that work consistently rather than a long-tail of broken, unsupported, ignored formulae.
```